specification/cognitiveservices/data-plane/Personalizer/preview/v1.0/Personalizer.json

```diff
@@ -0,0 +1,1148 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1.0",
+    "title": "Personalizer Client",
+    "description": "Personalizer Service is an Azure Cognitive Service that makes it easy to target content and experiences without complex pre-analysis or cleanup of past data. Given a context and featurized content, the Personalizer Service returns your content in a ranked list. As rewards are sent in response to the ranked list, the reinforcement learning algorithm will improve the model and improve performance of future rank calls."
+  },
+  "paths": {
+    "/personalizer/v1.0/configurations/service": {
+      "get": {
+        "tags": [
+          "Configuration"
+        ],
+        "summary": "Get the service configuration.",
+        "operationId": "GetServiceConfiguration",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfiguration"
+            },
+            "examples": {
+              "application/json": {
+                "rewardWaitTime": "00:10:00",
+                "defaultReward": 0.0,
+                "rewardAggregation": "earliest",
+                "explorationPercentage": 0.2,
+                "modelExportFrequency": "00:05:00",
+                "logRetentionDays": 7
+              }
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Configuration"
+        ],
+        "summary": "Update the service configuration.",
+        "operationId": "UpdateServiceConfiguration",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "config",
+            "in": "body",
+            "description": "The personalizer service configuration.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceConfiguration"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfiguration"
+            },
+            "examples": {
+              "application/json": {
+                "rewardWaitTime": "00:10:00",
+                "defaultReward": 0.0,
+                "rewardAggregation": "earliest",
+                "explorationPercentage": 0.2,
+                "modelExportFrequency": "00:05:00",
+                "logRetentionDays": 7
+              }
+            }
+          },
+          "400": {
```

Comment:
Only `default` return status code is supported for error status codes

Reply:
Sorry I dint understand. This is a bad request error. What code do you want to be used here ?

Reply:
Swagger can only handle successful status codes (200/201/202... and so on).
We do not support means to distinguish between error and successful status codes. All the error status codes are simply described by the `default` response. Take a look at [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/stable/2015-05-01/capabilities.json#L48) example

Reply:
fixed